app/views/tmdb/two_actor_search.html.erb

```diff
@@ -1,46 +1,33 @@
 <% content_for(:title, "Search by Actors") %>
 
-<h1>Find Common Movies Between 2 Actors</h1>
-
-
-<% if !@actor.present? && !@actor2.present? && !@not_found.present? %>
-
-<p>Enter two actors and see the movies they've been in together.</p>
-
+<h1>Find Common Movies Between Actors</h1>
```

Comment:
The main changes to this erb are around reducing the instance variable to a single `@results` object that has attributes. Though we're a few months out from our original mission of reducing instance variables in controllers/views/etc, here's where the rubber meets the road. :)

Reply:
> rubber meets the road
🍑